Pearson-Advance/openedx-themes
# PADV-237 - Update CertPREP home page

## Ticket

https://agile-jira.pearson.com/browse/PADV-237

## Description
 
This PR is intended to update the Cert PREP homepage according to the provided [mockup](https://agile-jira.pearson.com/secure/attachment/4330117/Pearson%20CertPREP%20landing%20page%20v2-2.pdf).

## Screenshots

![Screenshot from 2022-11-09 19-36-48](https://user-images.githubusercontent.com/62396424/200972970-414f999e-4971-4141-80c4-f9985e9d73a4.png)
![Screenshot from 2022-11-09 19-37-13](https://user-images.githubusercontent.com/62396424/200973010-899caf46-0d5e-411b-8ea9-a50c2a12cea3.png)
![Screenshot from 2022-11-09 19-37-28](https://user-images.githubusercontent.com/62396424/200973020-3b1efbc0-d349-4f65-aa19-8d2f5113a869.png)
![Screenshot from 2022-11-10 20-46-09](https://user-images.githubusercontent.com/62396424/201244626-24949e0e-9744-418e-b91c-f0d690bd9ae7.png)
![Screenshot from 2022-11-10 20-45-52](https://user-images.githubusercontent.com/62396424/201244633-21f4ca27-9c3f-43f4-9187-6aa02722e153.png)

## Type of Change

- [x] Update pearson-vue-theme homepage according to the mockup.
- [x]  Video modal

## Testing:

- Check out this branch.
- Enable Comprehensive theming: https://edx.readthedocs.io/projects/edx-installing-configuring-and-running/en/latest/configuration/changing_appearance/theming/enable_themes.html
- Compile assets: `paver update_assets --themes=pearson-vue-theme`
- Assign **pearson-vue-theme** to a new site.

## Reviewers
